### PR TITLE
feat(golang): add beta support

### PR DESCRIPTION
**Pull request by cfabianski**
## Description

Add Beta support for Golang

## Related

#1302 

## Checklist

- [ ] I've added test coverage that shows my fix or feature works as expected.
- [ ] I've updated or added documentation if required.
- [ ] I've included usage information in the description if CLI behavior was updated or added.
- [ ] PR title follows [Conventional Commits](https://www.conventionalcommits.org/) format

### CODE REVIEW

File: internal/util/progressbar/progressbar.go [hunk 1]
```diff
@@ -18,8 +18,9 @@ func GetProgressBar(filesLength int, config settings.Config, display_type string
 		progressbar.OptionOnCompletion(func() {
 			output.ErrorWriter().Write([]byte("\n")) //nolint:all,errcheck
 		}),
-		progressbar.OptionShowIts(),
-		progressbar.OptionSetItsString(display_type),
+		// progressbar.OptionShowIts(),
+		// progressbar.OptionSetItsString(display_type),
+		// progressbar.OptionClearOnFinish(),
```
Comment (didroe) [inline internal/util/progressbar/progressbar.go]: Is it intentional to include this change?

**Review by didroe — APPROVED** [review]
I'm not sure about the language template. I like the idea, but I worry it will be hard to keep it up-to-date - my thinking is that something not used by the product will likely slowly diverge from the real languages, even with the best effort to apply changes to it

**Comment by cfabianski** [comment]
> I'm not sure about the language template. I like the idea, but I worry it will be hard to keep it up-to-date - my thinking is that something not used by the product will likely slowly diverge from the real languages, even with the best effort to apply changes to it

Will delete the template after I'm done with Python. I agree.

File: internal/util/progressbar/progressbar.go (hunk 1 above)
Comment (cfabianski) [reply internal/util/progressbar/progressbar.go]: I will do it as a separate PR

**Pull request merged**